.github/workflows/deploy-fashion.yaml

```diff
@@ -0,0 +1,25 @@
+name: deploy fashion
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "demos/fashion-ecommerce/data/filter-config.csv"
+      - "demos/fashion-ecommerce/voice-interface.yaml"
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: |
+          npm install
+          npm run preprocess
```

Comment:
Using the pnpm caching and rush install as in deploy-demos.yaml would probably speed this up.